### PR TITLE
Completion for docker-compose

### PR DESCRIPTION
**Pull request by ilkka**
This change adds not-perfect-but-okay completion for docker-compose.

### CODE REVIEW

File: src/_docker-compose [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  __services
+}
+
+# common args
+_arguments \
+  '--verbose:Show more output' \
+  '--version:Print version and exit' \
+  '--file:Specify an alternate compose file:__yml_files_in_current_dir' \
+  '--project-name:Specify an alternate project name:()' \
+  '*:: :->command'
```
Comment (hhatto) [inline src/_docker-compose]: You might want to change the following line.

```
_arguments \
  '--verbose[Show more output]' \
  '--version:Print version and exit' \
  '--file[Specify an alternate compose file]:__yml_files_in_current_dir' \
  '--project-name[Specify an alternate project name]:args' \
  '*:: :->command'
```

**Comment by ilkka** [comment]
So finally got around to do this :D thanks for the feedback.

**Comment by rosstimson** [comment]
:+1:  Very useful, especially as `docker-compose` is a lot longer than `fig` and I don't want to have to add yet another alias.

**Comment by hhatto** [comment]
Sorry for late reply.
this patch is merged.

> :+1: Very useful, especially as docker-compose is a lot longer than fig and I don't want to have to add yet another alias.

Please send another pull request :-)

**Pull request merged**